ui/src/reusable_ui/components/wizard/WizardProgressBar.scss

```diff
@@ -0,0 +1,54 @@
+@import 'src/style/modules/influx-colors';
+@import 'src/style/modules/variables';
+
+.progress-bar {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+  margin: 20px 0;
+}
+
+.progress-button {
+  position: relative;
+  background-color: transparent;
+  border: none;
+  cursor: pointer;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  .progress-title {
+    font-size: $ix-text-tiny;
+    color: $g12-forge;
+    opacity: 0;
+    position: absolute;
+    white-space: nowrap;
+    margin-top: -$ix-marg-b;
+    font-weight: 600;
+  }
+}
+
+.progress-button:hover {
+  .progress-title {
+    color: $g12-forge;
+    opacity: 1;
+    margin-top: -$ix-marg-c;
+    transition: all .5s;
+  }
+}
+
+.checkmark:before {
```

Comment:
I suggest assigning colors to a class you can add to the element, for example `<span className="icon checkmark icon--green" />`. The `:before` pseudo element will inherit color from the `<span>` so it doesn't need to be so specific.